core/egov-localization/src/main/java/org/egov/web/controller/MessageController.java

```diff
@@ -79,9 +81,35 @@ private MessagesResponse createResponse(List<org.egov.domain.model.Message> doma
 		return getSuccessResponse(messages);
 	}
     
+    @PostMapping(value = "/_update")
+	@ResponseBody
+	public ResponseEntity<?> update(@RequestBody @Valid final NewMessagesRequest messagesRequest,
+			final BindingResult bindingResult) {
+		if (bindingResult.hasErrors()) {
+			throw new InvalidCreateMessageRequest(bindingResult.getFieldErrors());
+		}
+		logger.info("Update Message Request:" + messagesRequest);
+		messageService.createMessage(messagesRequest);
+		final List<Message> messages = messagesRequest.getMessages();
+		return getSuccessResponse(messages);
+	}
+    
+    @PostMapping(value = "/_delete")
+	@ResponseBody
```

Comment:
@darshannagesh Given this class has a RestController annotation this annotation is not needed.